core/psh/psh.c

```diff
@@ -1077,8 +1251,11 @@ static int psh_run(void)
 			psh_cat(argc, argv);
 		else if (!strcmp(argv[0], "exec"))
 			psh_exec(argc, argv);
-		else if (!strcmp(argv[0], "exit"))
+		else if (!strcmp(argv[0], "exit")) {
+			if(!exitable)
```

Comment:
By space do you mean one empty line after `if(!exitable)` or is there some place in that if block that need literally `" "` space? Because, as stated by you before :), there are places in code where `" "` improve readibility - thats why i ask.

Reply:
According to our coding convention there must be space between the keyword token and the opening brace token. You repeatedly ignore this rule, please fix it also in other places.

Reply:
> 
> 
> By space do you mean one empty line after `if(!exitable)` or is there some place in that if block that need literally `" "` space? Because, as stated by you before :), there are places in code where `" "` improve readibility - thats why i ask.

It should be `if (!exitable)`